azure-pipelines.yml

```diff
@@ -54,19 +54,25 @@ jobs:
   # just pulling in gfortran
   - script: brew install gcc
     displayName: 'make gfortran available on mac os vm'
+  # we currently seem to use homebrew for openblas in wheels
+  # infrastructure; at the time of writing this means
+  # openblas 0.3.3 on OSX is used vs. Linux clearly targeted at 0.3.0
```

Comment:
Can we clarify if this is for user install via Homebrew, in `numpy-wheels` for our releases, or both? IIRC it's Homebrew only, and our wheels have 0.3.0?

Reply:
I think I'm confused because, as noted in details drop-down above, I see [`brew install openblas`](https://github.com/matthew-brett/multibuild/blob/devel/library_builders.sh#L99) in the `multibuild` (submodule) repo, but I guess that is maybe not the repo I should be looking at.

Reply:
the `on_openblas.rst` in numpy-wheels suggests I may need to check another repo instead: https://github.com/MacPython/openblas-libs

Reply:
It's a bit unclear indeed.

- For Windows there's hardcoded SHA's of the OpenBLAS used that can be checked (in `appveyor.yml`).
- Both the NumPy and SciPy builds use an environment variable for Linux and macOS it looks like: https://github.com/MacPython/numpy-wheels/blob/master/env_vars.sh#L2, https://github.com/MacPython/scipy-wheels/blob/master/env_vars.sh#L2 (bothset to `v0.3.0` currently)

Reply:
Here is an example of changing the version that is used: https://github.com/MacPython/numpy-wheels/commit/3d0da6b6acac050e67509ed6a469dba1602b1890

Reply:
Ralf was right here -- I now correctly pull in the MacPython pre-built OpenBLAS binary in the workflow. Marking as resolved.